app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Employer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace BcGov.Fams3.SearchApi.Contracts.Person
+{
+   public  class Employer
+    {
+        [Description("The name of the employer")]
+        public string Name { get; set; }
+
+        [Description("The name of the owner of the employer")]
+        public string OwnerName { get; set; }
+
+        [Description("The phone number of the employer")]
+        public string PhoneNumber { get; set; }
+
+        [Description("The extension to the phone number of the employer")]
+        public string PhoneExtension { get; set; }
+
+        [Description("The Fax number of the employer")]
+        public string FaxNumber { get; set; }
+
+        [Description("The extension to the fax number of the employer")]
+        public string FaxExtension { get; set; }
+
+        [Description("The address of the employer")]
+        public Address Address { get; set; }
```

Comment:
shall we do the same for Phone? Maybe dynamics team can provide more input.

Reply:
I don't understand.. what should we do for phone ?

Reply:
I mean if we use Address class, for phonenumber and extension, we could also use Phone class

Reply:
Oh I don't recommend ... phone is a lot more that just number and extension. The address attached to Employer is the same as main address except for type which can be defaulted.

Reply:
sounds good.